apitrackingmore.py

```diff
@@ -103,14 +104,20 @@ def formato_obj(json, carrier, code, retries):
         situacao = evento['StatusDescription']
         observacao = evento['checkpoint_status']
         try:
-            observacao = 'Código novo: <code>' + geartrack.getcorreioscode(carrier, code) + '</code>'
+            codigo_novo = geartrack.getcorreioscode(carrier['code'], code)
+            if codigo_novo not in str(mensagem):
+                msg_codigo_novo = ('Data: {}' +
+                    '\nNovo código do pacote: <code>{}</code>'
+                ).format(data, codigo_novo)
         except:
             pass
         mensagem = ('Data: {}' +
             '\nSituacao: <b>{}</b>' +
             '\nObservação: {}'
         ).format(data, situacao, observacao)
         stats.append(mensagem)
+        if msg_codigo_novo != False:
+            stats.append(msg_codigo_novo)
```

Comment:
:)
```python
if msg_codigo_novo:
```

Reply:
Prefiro explícito..
Tá no Zen do Python, inclusive.. hehe

Reply:
> Don't compare boolean values to True or False using ==.

https://www.python.org/dev/peps/pep-0008/

> Readability counts.

Zen do Python :)

Reply:
hmm.. Blz
Mas então qualquer coisa que não for `False` vai entrar nesse if, certo?

Reply:
```python
if variável:
    ...
```
Só entra no `if` se:
- `variável` não for igual a `False`;
- `variável` não for igual a `None`;
- `variável` não for uma sequencia vazia: túplas `()` ou  listas`[]` ou strings `""` ou dicionários `{}`;
- `variável` não for igual a zero: `0` ou `0.0`;

Existe mais dois casos com métodos mágicos, `__bool__()` e `__len__()`, mas que não vem ao caso agora.